apps/folderlaunch/app.ts

```diff
@@ -252,6 +252,15 @@
     }
   }
 
+  /**
+   * Vibrate the watch if vibration is enabled
+   */
+  let buzz = (length: number | undefined = undefined, strength: number | undefined = undefined) => {
```

Comment:
If you like, we could avoid check inside the call:

```typescript
let buzz = config.disableVibration ? () => {} : Bangle.buzz;
```

Reply:
I did not know you could do that with javascript, thanks!